frontend/projects/upgrade/src/app/core/experiments/store/experiments.model.ts

```diff
@@ -228,8 +248,15 @@ export interface ExperimentState extends EntityState<Experiment> {
   allExperimentNames: {};
   contextMetaData: {};
   updatedStat?: IExperimentEnrollmentDetailStats;
+  isAliasTableEditMode: boolean;
+  aliasTableEditIndex: number | null;
```

Comment:
I let this be able to be `null` when `isAliasTableEditMode` is false, as there is no row-index being edited at that time.